mitosheet/mitosheet/code_chunks/empty_code_chunk.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+from typing import List, Optional
+from mitosheet.code_chunks.code_chunk import CodeChunk
+
+
+class EmptyCodeChunk(CodeChunk):
+    """
+    A code chunk that generates no code, but has a title. 
```

Comment:
TODO: Understand what this means. What does "has a title" mean? I can't think of an example where we don't want to generate code, but we want to display info about the step someplace else.

Reply:
Oh! Well the graphing steps are like that, I suppose!